config/boot.rb

```diff
@@ -56,6 +51,11 @@
   Config.setting_files(File.expand_path(__dir__), environment)
 )
 
+# Load local environment configuration
```

Comment:
Moved, else `Settings` is not available when loading `config/environments/test.rb`.